File: Android-Core/03-Components/RecyclerView/RCLoadAndRefresh/03refreshheader/src/main/java/top/iqqcode/refreshheader/NewActivity.java
package top.iqqcode.refreshheader;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

public class NewActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_new);

        Intent intent = getIntent();
        String itemName = intent.getStringExtra("itemName");
        TextView textView = findViewById(R.id.activityText);
        textView.setText(itemName);
    }
}

File: Android-Core/03-Components/RecyclerView/RecyclerViewGuiGu04/app/src/main/java/top/iqqcode/recyclerviewguigu04/NewActivity.kt
package top.iqqcode.recyclerviewguigu04

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import top.iqqcode.recyclerviewguigu04.databinding.ActivityNewBinding


class NewActivity : AppCompatActivity() {

    private lateinit var binding: ActivityNewBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityNewBinding.inflate(layoutInflater)
        var rootView: View = binding.root
        setContentView(rootView)

        binding.mNewText.text = intent.getStringExtra("position")
        binding.mNewText.text = intent.getStringExtra("imageClick")
    }
}